kickstarts/partials/post/source_setup.ks.erb

```diff
@@ -14,6 +14,10 @@ ssui_root="$repos_root/manageiq-ui-self_service"
 # Symlink manageiq to the vmdb root so that the SSUI can build into it
 ln -vs $vmdb_root $manageiq_root
 
+# Replace the log directory with the larger logvol
+rm -rf $vmdb_root/log
+ln -vs $app_root/vmdb_log $vmdb_root/log
```

Comment:
I like @geertj idea of increasing `/var/log` and  then symlink `/var/www/miq/vmdb/log` to a subdirectory below it.

This will require updates to `kickstarts/partials/post/source_setup.ks.erb` that will look something like:

```bash
# Replace the log directory with the larger logvol
rm -rf $vmdb_root/log
mkdir -p /var/log/vmdb_log
ln -vs /var/log/vmdb_log $vmdb_root/log
```

I'll make and test these updates. Once I get it working I'll update this PR and remove `WIP` 

Thank you! @geertj, @Fryguy and @carbonin and @simaishi!